config/rbac/role.yaml

```diff
@@ -20,13 +20,39 @@ rules:
 - apiGroups:
   - deployments.plural.sh
   resources:
-  - healthconverts/finalizers
+  - customhealths/finalizers
   verbs:
   - update
 - apiGroups:
   - deployments.plural.sh
   resources:
-  - healthconverts/status
```

Comment:
Should this change? To verify I would rerun all codegen targets from Makefile.

Reply:
idk I didnt' change it, I think that's the name now isn't it? maybe it wasn't updated before on main as this config dir isn't really used as a base for the chart